specs/MultiProfile.md

```diff
@@ -169,21 +169,27 @@ void ScenarioCookieManagement::DeleteAllCookies()
 ### Delete profile
 
 ```cpp
-HRESULT AppWindow::DeleteProfile(ICoreWebView2Controller* controller)
+HRESULT AppWindow::DeleteProfile(ICoreWebView2* webView2)
 {
-    wil::com_ptr<ICoreWebView2> coreWebView2;
-    CHECK_FAILURE(controller->get_CoreWebView2(&coreWebView2));
-    auto webview7 = coreWebView2.try_query<ICoreWebView2_7>();
-    if (webview7)
-    {
-        wil::com_ptr<ICoreWebView2Profile> profile;
-        CHECK_FAILURE(webview7->get_Profile(&profile));
-        auto profile2 = profile.try_query<ICoreWebView2StagingProfile4>();
-        if (profile2)
-        {
-            CHECK_FAILURE(profile2->Delete());
-        }
-    }
+    wil::com_ptr<ICoreWebView2Profile> profile;
+    CHECK_FAILURE(webView2->get_Profile(&profile));
+    CHECK_FAILURE(profile2->Delete());
+}
+
+EventRegistrationToken m_profileDeletedEventToken = {};
+void AppWindow::AddProfileDeleted(ICoreWebView2* webView2)
+{
+    CHECK_FAILURE(webView2->add_ProfileDeleted(
```

Comment:
To better match what we expect end devs to write, we would expect to see the add_ProfileDeleted to be in some sort of initialization method. And to not have a call to the remove_ProfileDeleted method. Please also make matching changes to the C# sample code.

Reply:
Ok, I will update it.